### PR TITLE
#137258429 Setup up Heroku instance of CFH

### PR DESCRIPTION
**Pull request by andela-ooyewole**
#### What does this PR do?
It aids the deployment of the application to a staging and production server(Heroku), by keeping database access information private for development environments.
#### Description of Task to be completed?
Install and configure 'dotenv' application to manage environment variables. Database authentication in particular.
#### How should this be manually tested?
This is tested by running a cloned and installed copy of this repo and expecting it to fail as a result of the database authentication credentials being undefined, but which when set locally, make the app run successfully.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
Setup up Heroku instance of CFH
#### Screenshots (if appropriate)
None
#### Questions:
None

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /**
  * Module dependencies.
  */
+require('dotenv').config();
```
Comment (q-ode) [inline server.js:4]: Let's have this along with all the other modules we already require.

**Review by q-ode — CHANGES_REQUESTED** [review]
Great Job!

Let's have a `.env.sample` file specifying the environment variables required.

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// replace <> with database server url to view on server, or 'localhost' to view on port 3000
+
+MONGOHQ_URL=<>
```
Comment (q-ode) [inline .env.sample]: Could you put a description for the variable in `<>`.

You wouldn't need the description above. It'll then look like:
`MONGOHG_URL=<URL to MondoDB Database>`

**Review by q-ode — CHANGES_REQUESTED** [review]
Nice one!

File: .env.sample (hunk 2 above)
Comment (andela-ooyewole) [reply .env.sample]: Okay I will do that. Thanks.

**Review by q-ode — APPROVED** [review]

**Pull request merged**